### DISCUSSION TITLE
Después de la Intro a Front

### DISCUSSION THREAD

**Discussion by RafBD** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#42] [Q&A]
Hola, he estado muy ocupado estos días y creo que me he atrasado un poco. Quería saber si ya tenemos que hacer algún ejercicio. 
Ya termine de leer todo sobre la introducción a FrontEnd y apenas estoy viendo la sesión del miércoles. 
Mi pregunta es ¿debemos de hacer algo después de leer? ¿hay algún ejercicio?

Gracias. :D

**Comment by JLuisPrz** [comment]
Hola, tenemos que hacer esta practica: #21

**Reply by RafBD** [reply]
Ah ok, pero no me queda muy claro que debemos de hacer? :(

**Reply by JLuisPrz** [reply]
En este link https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/01%20-%20INTRO/practicas nos dan un caso con el cual debemos de realizar los pasos que nos pide.
De cualquier modo mañana en el live supongo que se resolverán muchas dudas al respecto.

**Reply by RafBD** [reply]
Bueno, pues esperare mañana para intentar entender, gracias :)

**Reply by NayGL** [reply]
X2, estoy igual ya termine de leer, ya voy por el segundo vídeo y no termino de entender que hacer :(
